ecocode-rules-specifications/src/main/rules/EC14/EC14.json

```diff
@@ -0,0 +1,20 @@
+{
+    "title": "Do not disable idle timer, unless absolutely necessary.",
+    "type": "CODE_SMELL",
+    "status": "ready",
+    "remediation": {
+      "func": "Constant\/Issue",
+      "constantCost": "5min"
+    },
+    "tags": [
+      "eco-design",
+      "environment",
+      "sobriety",
+      "ecocode"
+    ],
+    "defaultSeverity": "Major",
+    "compatibleLanguages": [
```

Comment:
I don't think it's useful to set this property.

Additionally, JSON in rule folder level is supposed to be language agnostic.
For language specificities, it would be necessary to be able to manage a JSON override with a JSON file placed in the corresponding subfolder language directory.

The ability to overload JSON metadata is currently not implemented (but could be if necessary).

Reply:
To be honest, I followed the template of the others rules at the time (late August) and almost all of them had this property and still are (eg: [EC11](https://github.com/green-code-initiative/ecoCode/blob/main/ecocode-rules-specifications/src/main/rules/EC11/EC11.json#L15), [EC12](https://github.com/green-code-initiative/ecoCode/blob/main/ecocode-rules-specifications/src/main/rules/EC12/EC12.json#L15), etc.).

So what should I do, remove or leave it?

Reply:
OK, leave it like that.
We improve metadata in future commits if needed